ykrt/src/trace/mod.rs

```diff
@@ -161,6 +162,15 @@ impl IRTrace {
         (di_tmp, di_fd, di_tmpname_c)
     }
 
+    #[cfg(test)]
```

Comment:
Hah, filth :)

Reply:
Yep, this is horrendous!

Reply:
(sorry, that was meant for `llvmbc_section`)